bin/hdfcoinc/pycbc_page_foreground

```diff
@@ -71,7 +71,10 @@ if args.output_file.endswith('.xml') or args.output_file.endswith('.xml.gz'):
         err_msg += "trigger lists with --single-detector-triggers."
         raise ValueError(err_msg)
 
-if h_num_rm >= 0:
+# Insure that the user uses a coinc trigger file that is hierarchical
+# removal compatible. Also default to regular behavior if hierarchical
+# removals were not performed.
+if h_num_rm >= 0 and h_iterations is not None and h_iterations!=0:
```

Comment:
What if the file is missing the hierarchical data completely? Do you catch that case?